spec/parts/linux/cloud-init/artifacts/cse_config_spec.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+Describe 'cse_config.sh'
```

Comment:
Im a bit confused, what sort of test is that? I mean the syntax

Reply:
tests for that particular file, every test which tests funcs from that file will need to be in this describe block afaik